client/src/views/Simulation/Simulation.vue

```diff
@@ -193,12 +184,46 @@
       }));
       this.setSimParameters({ parameters, count: 1 });
     }
+
+    async fetchVariables (): Promise<void> {
```

Comment:
maybe this should be renamed to `initializeX`, all 3 of them here. note to self

Reply:
If you prefer, I use the word `fetch` because we are getting them from the Donu API.